askama/src/filters/mod.rs

```diff
@@ -342,14 +342,14 @@ pub fn wordcount<T: fmt::Display>(s: T) -> Result<usize> {
 }
 
 #[cfg(feature = "markdown")]
-pub fn markdown<E, S>(
+pub fn markdown<E,S>(
```

Comment:
You've left a bunch of trivial changes in this PR, please actually revert the changes to the previous state.